osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs

```diff
@@ -79,15 +79,15 @@ public override Anchor Origin
             {
                 var origin = base.Origin;
 
-                if (FlipH)
+                if ((FlipH || VectorScale.X < 0) && !(FlipH && VectorScale.X < 0))
```

Comment:
This can be simplified to one XOR operation, such that both `true`s become `false`, but unsure whether that reads better in C# world:
```suggestion
                if (FlipH ^ (VectorScale.X < 0))
```

Reply:
Would prefer that because I struggled reading the original at first. Can be split into two lines like...
```csharp
bool flipH = FlipH;
flipH ^= VectorScale.X < 0; // Either flip or negative scale, but not both.
```

Reply:
I think a comment would be enough to explain it, how does this read:

![CleanShot 2022-03-14 at 04 33 21](https://user-images.githubusercontent.com/22781491/158090055-d6584ff8-2b15-4b3f-b9e4-97fd423ab3c2.png)

Reply:
Looks fine to me